packages/cosmwasm-stargate/src/aminotypes.spec.ts

```diff
@@ -0,0 +1,337 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import {
+  MsgClearAdmin,
+  MsgExecuteContract,
+  MsgInstantiateContract,
+  MsgMigrateContract,
+  MsgStoreCode,
+  MsgUpdateAdmin,
+} from "@cosmjs/cosmwasm-launchpad";
+import { fromBase64, toAscii } from "@cosmjs/encoding";
+import { coins } from "@cosmjs/launchpad";
+import { AminoTypes } from "@cosmjs/stargate";
+import Long from "long";
+
+import { defaultTypes } from "./aminotypes";
+import { cosmwasm } from "./codec";
+
+type IMsgStoreCode = cosmwasm.wasm.v1beta1.IMsgStoreCode;
+type IMsgInstantiateContract = cosmwasm.wasm.v1beta1.IMsgInstantiateContract;
+type IMsgUpdateAdmin = cosmwasm.wasm.v1beta1.IMsgUpdateAdmin;
+type IMsgClearAdmin = cosmwasm.wasm.v1beta1.IMsgClearAdmin;
+type IMsgExecuteContract = cosmwasm.wasm.v1beta1.IMsgExecuteContract;
+type IMsgMigrateContract = cosmwasm.wasm.v1beta1.IMsgMigrateContract;
+
+describe("AminoTypes", () => {
+  describe("toAmino", () => {
+    it("works for MsgStoreCode", () => {
+      const msg: IMsgStoreCode = {
+        sender: "cosmos1pkptre7fdkl6gfrzlesjjvhxhlc3r4gmmk8rs6",
+        wasmByteCode: fromBase64("WUVMTE9XIFNVQk1BUklORQ=="),
```

Comment:
Very nice we now fully control the field name style, as it should be for protobuf